docs/Guide/getting-started/creating-a-basic-command.mdx

```diff
@@ -17,6 +17,13 @@ Note that this doesn't create subcommands. We'll cover those in
 
 :::
 
+:::info
+
+Note that by default Sapphire does not load the listeners required for processing message commands. You will need to construct your `SapphireClient` with
+the `loadMessageCommandListeners` option set to `true`. See more information about the [client options](https://www.sapphirejs.dev/docs/Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners).
```

Comment:
```suggestion
Note that by default Sapphire does not load the listeners required for processing message commands. You will need to construct your `SapphireClient` with
the `loadMessageCommandListeners` option set to `true`. For more information see the [`SapphireClientOptions#loadMessageCommandListeners`][load-message-ecommands-option] interface.
```

Add to the bottom of this file:

```
[load-message-ecommands-option] ../../Documentation/api-framework/interfaces/SapphireClientOptions#loadmessagecommandlisteners
```

Reply:
@AnotherBrynAlt You didn't add the link to the bottom of the file as mentioned above